tests/basic.py

```diff
@@ -233,3 +248,35 @@ def verify(self, context, state):
 @pytest.mark.parametrize("type", SSH_KEY_TYPES)
 def test_ssh_resident(device, type) -> None:
     TestSshResident(type).run(device)
+
+
+def test_opcard_p256(device):
+    out = subprocess.getoutput("opgpcard list")
+    card_id = out.split("\n")[1].strip()
+
+    p = spawn(f"opgpcard system factory-reset --card {card_id}")
+    p.expect_exact(f"Resetting Card {card_id}")
+    p.expect(EOF)
```

Comment:
I think we can derive the card ID from `device.serial`.  We should check that only this device is connected so that we don’t accidentally reset some other connected OpenPGP card.

Reply:
done